include/kodlab_mjbots_sdk/interfaces.h

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
We may want to do a check that we have run this file and no the individual interfaces. 

in particular that might look like
HERE:
`#define _interfaces_defined`
MjBotsXXXXINterface.h:
`#ifndef _interfaces_defined
    ERRORORSOMETHING`

Reply:
Or maybe something at the CMAKE level 
Error if this is included